src/index.js

```diff
@@ -1,12 +1,22 @@
 import express from "express";
 import comments from "./comments";
 import products from "./products";
-import vehicle from "./vehicles";
+import vehicles from "./vehicles";
 import contacts from "./contacts";
+import bodyParser from "body-parser";
+import contactRoutes from "./routes/contactRoutes";
+import commentRoutes from "./routes/commentRoutes";
+import productRoutes from "./routes/productRoutes";
+import vehicleRoutes from "./routes/vehicleRoutes";
 
```

Comment:
Looks like I forgot to put this in the instructions, but we need to add the mongoose code to connect to MongoDB. That goes in index.js